utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -552,4 +553,4 @@
 "zope 2.0": ZPL-2.0
 "zope 2.1": ZPL-2.1
 "zope license": ZPL-2.1
-"zope v2.1": ZPL-2.1
\ No newline at end of file
+"zope v2.1": ZPL-2.1
```

Comment:
The fix-up went into the wrong commit.

Reply:
I helps if you do a `git push -f` after you fix up things :-(